src/Queue.ts

```diff
@@ -83,7 +83,7 @@ export class Queue {
   }
 
   public async listen<T>(callback: (data: T, message?: amqp.ConsumeMessage) => Promise<boolean>) {
-    if (!this.options.topic) {
+    if (!this.options.topics) {
```

Comment:
this.options.topics.length === 0

array converte para "true", mesmo array vazio

Reply:
Feito, seria bom no futuro escrevermos alguns testes pra pegar esses casos